docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -68,7 +68,15 @@ If you are reading this topic with the questions listed below, click them to jum
 
     >**NOTE**: For more information of the preceding services, see [Nebula Graph architecture](../1.introduction/3.nebula-graph-architecture/1.architecture-overview.md).
 
-4. Connect to Nebula Graph. For detailed instructions, see [Use nebula-console to connect to Nebula Graph](3.connect-to-nebula-graph.md/#use_nebula-console_to_connect_to_nebula_graph).
+4. Connect to Nebula Graph.
+
+    ```bash
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
+    ```
+
+    ```bash
+    docker> nebula-console -u <user> -p <password> --address=graphd --port=3699
+    ```
```

Comment:
4. Use Nebula Console to connect to Nebula Graph. OR "Get access to Nebula Graph with Nebula Console. "
 
```bash
    # Start a Nebula Console service in a Docker container? 
    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
    
    # Connect Nebula Console to Nebula Graph 
    docker> nebula-console -u <user> -p <password> --address=graphd --port=3699
    ```

according to this thread: https://discuss.nebula-graph.com.cn/t/topic/2209/15, I think maybe some comments should be added to explain these commands.